deploy/vagrant/bin/init.py

```diff
@@ -13,10 +15,19 @@
 import sys
 from sys import stderr
 import argparse
+import json
+import tempfile
+import errno
+
+import boto
+from boto import ec2
+import yaml
 
 def parse_args():
     parser = argparse.ArgumentParser()
     parser.add_argument('provider')
+    parser.add_argument('--is-aws-spot', action='store_true')
```

Comment:
Originally, wished to put all initialization to `init.py`, but now, only aws related initializations are put there. Yup, I will rename it to `init_aws.py` and remove the provider argument.